dapps/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.vue

```diff
@@ -48,7 +48,7 @@
         <div class="dapp-wrapper-sidebar" v-if="!onboarding && enableSidebar">
           <slot name="sidebar">
             <ul class="nav top-nav">
-              <li v-for="(route, index) in routes">
+              <li v-for="(route, index) in routes" v-bind:key="index">
```

Comment:
fixes the linter error, but using index a as key is IMO an anti pattern in vue.
Also if I also did it from time to time :D

Reply:
Fixed it, thanks